src/index/script/HeadsUpDisplay/PositionWidget/_PositionWidgetState.ts

```diff
@@ -0,0 +1,6 @@
+
+
+export default class PositionWidgetState {
+
```

Comment:
Is this meant to be blank?  If so - add a TODO comment or similar to communicate intent... but normally this should be ripped out.

Reply:
It is intended to be blank -- TypeScript wants a state and props when extending `React.Component`, so this is here just to satisfy that requirement. I might fill it in later if I need to. I'll make a comment explaining this.